scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Use the following reference information to understand the {sno} configurations required to deploy virtual distributed unit (vDU) applications in the cluster. Configurations include cluster optimizations for high performance workloads, enabling workload partitioning, and minimizing the number of reboots required post-installation.
+Use the following reference information to understand the {sno} configurations required to deploy virtual distributed unit (vDU) applications in the cluster. Configurations include cluster optimizations for high performance workloads, enabling workload partitioning, and minimizing the number of reboots required postinstallation.
```

Comment:
TIL re: IBM SG

Although it looks like in-repo there are 110 results for "post-installation" and (other than this new one) 0 hits on "postinstallation". 🤷 Just as far as considering consistency vs maybe correctness.

Reply:
https://github.com/openshift/openshift-docs/pull/67093